appveyor.yml

```diff
@@ -0,0 +1,11 @@
+version: '{build}'
+skip_tags: true
+image: Visual Studio 2017
+configuration: Release
+install:
+  - ps: mkdir -Force ".\build\" | Out-Null
+build_script:
+- ps: ./Build.ps1
+test: off
+artifacts:
+- path: artifacts/Serilog.*.nupkg
```

Comment:
Ideally this should be identical to: https://github.com/serilog/serilog-sinks-file/blob/dev/appveyor.yml - missing some of the publishing information (the whole file can be copied in exactly as-is).